script/input/11155111/tokenVotingModuleConfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "comment": "This is an example token voting module deployment on Sepolia.",
-  "factory": "0x6A97643633eafEEC00b7Ec4CE84269203645aaBC",
+  "factory": "0xFBE17545dffD75A92A5A72926AE581478973FE65",
   "llamaCore": "0xc68046794327490F953EA15522367FFBA0b64f86",
```

Comment:
Maybe we should make this the 0 address so it forces the user to provide their own llama core?

Reply:
Fair enough

Reply:
Done in [44b1418](https://github.com/llamaxyz/llama-periphery/pull/95/commits/44b141866abe12800d1609c4d184144149c6f549)